windows/jitsi-meet/src/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import {
+    RemoteControl,
+    setupScreenSharingForWindow,
+    setupAlwaysOnTopRender,
+    setupWiFiStats
+} from 'jitsi-meet-electron-utils';
+
+import { jitsiMeetDomain } from '../../../config.js';
+
+/**
+ * Jitsi Meet Window Component
+ */
+class JitsiMeetWindow extends React.Component {
+
+    /**
+     * Render function of component
+     * @return {ScriptElement}
+     */
+    render() {
+        return null;
+    }
+
+    /**
+     * Attach the script
+     */
+    componentDidMount() {
```

Comment:
Let's do it on WillMount.

Reply:
Are you sure?

**componentDidMount()**
https://reactjs.org/docs/react-component.html#componentdidmount
"If you need to load data from a remote endpoint, this is a good place to instantiate the network request."
Not the same thing. But similar maybe.

Reply:
You're right, let's keep it there. Later, when we add some form of loading indicator.